include/nuttx/signal.h

```diff
@@ -112,6 +112,68 @@ struct sigwork_s
 
 struct timespec;  /* Forward reference */
 
+/****************************************************************************
+ * Name: nxsig_ismember
+ *
+ * Description:
+ *   This function tests whether the signal specified by signo is a member
+ *   of the set specified by set.
+ *
+ * Input Parameters:
+ *   set - Signal set to test
+ *   signo - Signal to test for
+ *
+ * Returned Value:
+ *   1 (true), if the specified signal is a member of the set,
+ *   0 (OK or FALSE), if it is not, or
+ *  -1 (ERROR) if the signal number is invalid.
```

Comment:
Internal OS functions should never return -1 (ERROR), then should return a negated errno value.    In this case it should return -EINVAL:

[EINVAL]
    The signo argument is not a valid signal number, or is an unsupported signal number.

Reply:
```suggestion
 * -EINVAL if the signal number is invalid
```

Reply:
Done.